cmd/logout.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var logoutCmd = &cobra.Command{
+	Use:   "logout",
+	Short: "Log out of the active session",
```

Comment:
Maybe do: `Log out of the current OpenShift session` ? Make this OpenShift specific!